dev/generate-kernel-signatures.py

```diff
@@ -93,7 +93,8 @@
     "awkward_reduce_max",
     "awkward_reduce_min",
     "awkward_reduce_sum",
-    "awkward_reduce_sum_int32_bool_64" "awkward_reduce_sum_int64_bool_64",
+    "awkward_reduce_sum_int32_bool_64",
+    "awkward_reduce_sum_int64_bool_64",
```

Comment:
Yikes! Nice catch.

Reply:
That's what linters are for. :)

My hand rewrite of the template code is still slightly off, will have to find where things changed.